ios/KlaviyoReactNativeSdk.mm

```diff
@@ -72,9 +72,14 @@ - (NSDictionary *)constantsToExport {
     [KlaviyoBridge setPushToken: pushToken];
 }
 
+RCT_EXPORT_METHOD(getPushToken: (RCTResponseSenderBlock)callback) {
+    NSString *pushToken = [KlaviyoBridge getPushToken];
+    callback(@[pushToken]);
+}
+
 RCT_EXPORT_METHOD(setBadgeCount: (nonnull NSNumber *)count)
 {
-  [KlaviyoBridge setBadgeCount: count.intValue];
+    [KlaviyoBridge setBadgeCount: count.intValue];
```

Comment:
ran indent fix on this too

Reply:
I think when I open on jet brains it'll format it differently?

Reply:
hm lemme check, this also seemed more consistent with the rest of the file though.

Reply:
Yeah doesn't seem to change anything when I open in jetbrains either now